packages/hooks/use-throttle-render/index.ts

```diff
@@ -1,31 +1,55 @@
 import { onMounted, ref, watch } from 'vue'
+import { isNumber, isObject, isUndefined } from '@element-plus/utils'
 
+import type { SkeletonThrottle } from '@element-plus/components'
```

Comment:
Avoid import components content in hooks. It is recommended to export the `SkeletonThrottle` type here.

Reply:
This issue does not seem to be resolved

Suggest moving the type declaration of `SkeletonThrottle` to here

Reply:
Okay, I understand what you mean, I will also change the `SkeletonThrottle` field to `ThrottleType` to increase generality.